app/graphql/mutations/profile/associate_systems.rb

```diff
@@ -6,6 +6,8 @@ module Profile
     class AssociateSystems < BaseMutation
       graphql_name 'associateSystems'
 
+      enforce_rbac Rbac::POLICY_WRITE
```

Comment:
This might not be sufficient, system assignment is not a step included in policy creation. But that might be intentional, need to consult with PM.